src/Symfony/Component/Finder/Iterator/SortableIterator.php

```diff
@@ -83,15 +89,8 @@ public function __construct(\Traversable $iterator, $sort, bool $reverseOrder =
      */
     public function getIterator()
     {
-        if (1 === $this->sort) {
-            return $this->iterator;
```

Comment:
this early return prevented the `iterator_to_array` to array below, saving some memory in the process